src/main/scala/nl.knaw.dans.easy.dd2d/mapping/Language.scala

```diff
@@ -40,47 +40,37 @@ object Language extends BlockBasicInformation with DebugEnhancedLogging {
   )
 
   def toBasicInformationBlockLanguageOfFiles(node: Node): Option[JsonObject] = {
-    val isoLanguage = getISOLanguage(node)
-    isoLanguage match {
-      case "" =>
-        logger.error(s"Invalid controlled vocabulary term for 'Language of Files': '$isoLanguage'. Ignoring.")
-        None
-      case _ =>
+
+    iso639_2ToDataverse.get(node.text)
+      .map(isoLanguage => {
         val m = FieldMap()
         m.addPrimitiveField(LANGUAGE_OF_FILES_CV_VALUE, isoLanguage)
         m.addPrimitiveField(LANGUAGE_OF_FILES_CV_VOCABULARY, LANGUAGE_OF_FILES_CV_VOCABULARY_NAME)
         m.addPrimitiveField(LANGUAGE_OF_FILES_CV_VOCABULART_URL, LANGUAGE_CV_ISO_639_2_URL + node.text)
-        Some(m.toJsonObject)
-    }
+        m.toJsonObject
+      })
```

Comment:
`doIfNone` is missing here

Reply:
This one is not for a controlled vocabulary field.